app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,51 @@
+.header-part{
+  margin: 1.5em 0;
+}
+.head-logo{
+  float:left;
+}
+.top-nav{
+  float: right;
+}
+.top-nav1{
+  float: right;
+}
+.top-nav span{
+  display:none;
+}
+.top-nav1 span{
+  display:none;
+}
+.top-nav ul{
+  padding:0;
```

Comment:
Properties should be ordered margin, padding<br>Colon after property should be followed by one space